alembic/env.py

```diff
@@ -0,0 +1,93 @@
+import asyncio
+from logging.config import fileConfig
+
+from sqlalchemy import pool
+from sqlalchemy.engine import Connection
+from sqlalchemy.ext.asyncio import async_engine_from_config
+
+from alembic import context
+
+from src.database import Base
+from src.city.models import *
+from src.temperature.models import *
```

Comment:
Wildcard imports (e.g., `from src.city.models import *`) can lead to namespace pollution and make it difficult to track where certain functions or classes are coming from. It's generally better to import only the necessary components explicitly.